Sources/CubeFoundationSwiftUI/Text/TextStyle.swift

```diff
@@ -31,4 +32,26 @@ public struct TextStyle: Hashable {
         self.letter = letter
         self.underline = underline
     }
+
+    // MARK: - Helper
+
+    /// Make a `Font`
+    public var mappedFont: Font {
```

Comment:
Property `font` is already taken, could consider a small breaking change 🤔

Reply:
Property name would be `customFont` (or `fontName` depending on what we go for), then this computed property could be `font`. But is breaking 😬